test/src/main/java/com/microsoft/test/embeddedsocial/AsyncTest.java

```diff
@@ -0,0 +1,71 @@
+package com.microsoft.test.embeddedsocial;
```

Comment:
Should we add a copyright to these files?

Reply:
done.